tpot/base.py

```diff
@@ -232,7 +233,8 @@ def __init__(self, generations=100, population_size=100, offspring_size=None,
             A setting of 2 or higher will add a progress bar during the optimization procedure.
         disable_update_check: bool, optional (default: False)
             Flag indicating whether the TPOT version checker should be disabled.
-
+        progress_file: io.TextIOWrapper or io.StringIO, optional (defaul: sys.stdout)
+            Save progress content to a file.
```

Comment:
Could you please rename the `progress_file` to `log_file`? I think there are some log infos when `verbosity=3`.